liquidctl/driver/asetek_pro.py

```diff
@@ -0,0 +1,256 @@
+"""liquidctl drivers for sixth generation Asetek 690LC liquid coolers.
+
+Supported devices:
+
+- Corsair H100i PRO RGB, H115i PRO RGB or H150i PRO RGB (Experimental)
+
+Copyright (C) 2018–2020  Jonas Malaco and contributors
+
+Incorporates or uses as reference work by Kristóf Jakab, Sean Nelson
+and Chris Griffith.
+
+SPDX-License-Identifier: GPL-3.0-or-later
+"""
+
+import logging
+
+import usb
+
+from liquidctl.driver.asetek import Modern690Lc
+from liquidctl.driver.usb import UsbDriver
+from liquidctl.error import NotSupportedByDevice
+from liquidctl.keyval import RuntimeStorage
+from liquidctl.util import clamp, LazyHexRepr
+
+
+_LOGGER = logging.getLogger(__name__)
+
+_MAX_PROFILE_POINTS = 7
+_CRITICAL_TEMPERATURE = 60
+_HIGH_TEMPERATURE = 45
+_READ_LENGTH = 32
+_READ_TIMEOUT = 2000
+_WRITE_TIMEOUT = 2000
+
+_PRO_WRITE_ENDPOINT = 0x1
```

Comment:
Private constants don't need a prefix based on this module's name; `_WRITE_ENDPOINT` saves a few columns and keystrokes, and is just as clear.